players.sql

```diff
@@ -0,0 +1,29 @@
+CREATE TABLE players(
+id integer UNIQUE,
+last_name text,
+first_name text,
+position text ,
+jersery_number integer,
```

Comment:
You spelled _jersey_ incorrectly.

```suggestion
jersey_number integer,
```